Makefile

```diff
@@ -5,6 +5,7 @@ publish: ## publish all projects to pub.dev
 
 example_run_build_runner: ## Generate code for the example app inside the widget_driver folder.
 	bash scripts/run-example-run-build-runner.sh
+	make format
```

Comment:
It makes sense to format the newly generated code according to our formatting rules.